Makefile

```diff
@@ -35,19 +37,29 @@ help:
 	@echo
 	@echo 'Configuration options:'
 	@echo
-	@echo '  GITHUB_USER       Which GitHub user account to use for cloning? [default=anonymous]'
-	@echo '  MINIKUBE_DRIVER   Which Minikube driver to use? [default=kvm2]'
-	@echo '  MINIKUBE_PROFILE  Which Minikube profile to use? [default=minikube]'
-	@echo '  TIMECHECK         Checking frequency in seconds when bringing cluster up and down? [default=5]'
-	@echo '  TIMEOUT           Maximum timeout to wait when bringing cluster up and down? [default=300]'
-	@echo '  VENV_NAME         Which Python virtual environment name to use? [default=reana]'
-	@echo '  DEMO              Which demo example to run? [e.g. reana-demo-helloworld; default=several runable examples]'
+	@echo '  GITHUB_USER         Which GitHub user account to use for cloning? [default=anonymous]'
+	@echo '  MINIKUBE_DRIVER     Which Minikube driver to use? [default=kvm2]'
+	@echo '  MINIKUBE_PROFILE    Which Minikube profile to use? [default=minikube]'
+	@echo '  TIMECHECK           Checking frequency in seconds when bringing cluster up and down? [default=5]'
+	@echo '  TIMEOUT             Maximum timeout to wait when bringing cluster up and down? [default=300]'
+	@echo '  VENV_NAME           Which Python virtual environment name to use? [default=reana]'
+	@echo '  DEMO                Which demo example to run? [e.g. reana-demo-helloworld; default=several runable examples]'
+	@echo '  REANA_CLUSTER_MODE  REANA Cluster environment mode [stable|latest|dev, default=latest]'
+	@echo '  DEBUG               Should REANA run with autoreload and debugging capabilities? [default=false]'
```

Comment:
DEBUG=0 or DEBUG=1 might be more shell-oriented, since true/false seems more like Python oriented.

BTW do we want to enable DEBUG=1 for PROD configurations? Perhaps the value of DEBUG can depend on the value of `CLUSTER_CONFIG`, i.e. enabled for DEV and disabled otherwise?